src/pages/movieList.js

```diff
@@ -2,16 +2,24 @@
 
 import {createMovieCard} from '../components/card.js';
 import {renderPageLayout} from '../layouts/createLayout.js';
+import {getTopRatedFilms} from '../services/api.js';
 
 export function createMovieList() {
     const arrayWithCards = [];
 
-    for (let i = 0; i < 20; i++) {
-        arrayWithCards.push(createMovieCard());
-    }
+    getTopRatedFilms().then(result => {
+        
+        result.results.forEach(element => {
+            const movieInfo = {};
```

Comment:
@Mmarvinn You do not need this separate variable for passing parameters in the function. You can just write like this 
```js
arrayWithCards.push(createMovieCard({
  posterOfFilm: element.poster_path,
  filmName:  element.original_title,
  filmOverview:   element.overview,
}));
```

Reply:
fix